### PR TITLE
 check if is ok

### PR DESCRIPTION
**Pull request by mugijanvier**
@alptilev

### CODE REVIEW

File: seshdash/views.py [hunk 1]
```diff
@@ -267,11 +267,11 @@ def get_high_chart_data():
 
     # Getting climat conditions
 
-     high_cloud_cover = Site_Weather_Data.objects.filter(date__range=[five_day_past2,five_day_future2]).values_list('cloud_cover', flat=True).order_by('date')
+     high_cloud_cover = list(Site_Weather_Data.objects.filter(date__range=[five_day_past2,five_day_future2]).values_list('cloud_cover', flat=True).order_by('date'))
```
Comment (jlark) [inline seshdash/views.py]: Use also need to filter by site location. There can be multiple sites at different locations. Currently in your filter there is nothing that will limit the location.

File: seshdash/views.py [hunk 2]
```diff
@@ -267,11 +267,11 @@ def get_high_chart_data():
...
      context_high_data['high_cloud_cover']=high_cloud_cover
      # Getting climat Dates and the Pv Daily production
      # Getting  Dates  Site_Weather_Data is where i can find the date interval for dynamic initialization
-     high_date = Site_Weather_Data.objects.filter(date__range=[five_day_past2,five_day_future2]).values_list('date', flat=True).order_by('date')
```
Comment (jlark) [inline seshdash/views.py]: Same as above

**Comment by jlark** [comment]
see comments above

**Comment by jlark** [comment]
Hi Janvier 
Can you check my comments and get back to me.

Happy new year.

**Comment by mugijanvier** [comment]
hey,  sorry !! yes i will. i now at klab i will work on it!!!!

2016-01-03 13:40 GMT+02:00 Alp Tilev <notifications@github.com>:

**Pull request closed without merging**